src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const total = days * 40;
+
+  if (days >= 3 && days < 7) {
+    return total - 20;
+  } else if (days >= 7) {
+    return total - 50;
+  } else {
+    return total;
+  }
```

Comment:
what a magic numbers 🪄
lets create variables for discount amount and days and then use them
it is better for managing them from one place and for understanding what do they mean

Reply:
Also avoid such nesting `else if` conditions
This code is hard to read and maintain